filebeat/module/haproxy/http/test/haproxy.log-expected.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
This file should contain only the list of events (each event being the value of `_source`)